app/actions/navigation/index.js

```diff
@@ -181,3 +182,31 @@ export function goToModalAccountSettings() {
         }, getState);
     };
 }
+
+export function goToCreateChannel(channelType) {
+    return async (dispatch, getState) => {
+        closeDrawers()(dispatch, getState);
+        let type;
+        let route;
+        switch (channelType) {
+        case Constants.OPEN_CHANNEL:
+            type = NavigationTypes.NAVIGATION_PUSH;
```

Comment:
Why is the navigation different for both of these when they're using the same scene component?